client/app/components/accounts/server.cjsx

```diff
@@ -56,15 +56,8 @@ module.exports = AccountServer = React.createClass
 
 
     render: ->
-        options = ServersEncProtocols
-            .map (protocol) ->
-                value: protocol
-                label: t "server protocol #{protocol}"
-            .concat
-                value: 'none'
-                label: 'server protocol none'
-
-        <div className="content">
+        <div className="content server">
+            <h2>{@props._protocol}</h2>
```

Comment:
Why are you using `_`as a prefix? I think that if you really want this method to be private, you should remove it from ReactComponent or do nothing else than removing the prefix :)